.github/CONTRIBUTING.md

```diff
@@ -55,25 +55,25 @@ QIcon(const QString &fileName)
 QIcon(QIconEngine *engine)
 ```
 
-No `QLatin1String` - mentioned - so use `QStringLiteral` instead, which creates us a `QString()` at compile-time so at least creating the object is faster.
+No `QLatin1String` - mentioned - so use `qsl` instead, which creates us a `QString()` at compile-time so at least creating the object is faster.
 
-([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/),
- [additional reading](https://woboq.com/blog/qstringliteral.html))
+([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qsl/),
+ [additional reading](https://woboq.com/blog/qsl.html))
 
-## Do not use ``QStringLiteral("")``
+## Do not use ``qsl("")``
 
-Prefer ``QString()`` over ``QStringLiteral("")`` for  for empty strings - the default constructor 
+Prefer ``QString()`` over ``qsl("")`` for  for empty strings - the default constructor
 for QString is cheaper in terms of both instructions and memory.
 
-([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
+([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qsl/))
 
 ## Avoid duplicated QStringLiterals
 
 Avoid having multiple QStringLiterals with the same content. For plain literals and QLatin1String, compilers
-try to consolidate identical literals so that they are not duplicated. For QStringLiteral, identical strings
+try to consolidate identical literals so that they are not duplicated. For qsl, identical strings
 cannot be merged.
 
-([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
+([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qsl/))
```

Comment:
One place that should not have been *replaced*? 😜

Reply:
Fixed.